pkg/experimental/epic/epic.go

```diff
@@ -107,13 +107,23 @@ func VerifyTimestamp(timestamp time.Time, epicTS uint32, now time.Time) (err err
 // If the same buffer is provided in subsequent calls to this function, the previously returned
 // EPIC MAC may get overwritten. Only the most recently returned EPIC MAC is guaranteed to be
 // valid.
-// @ trusted
-// @ requires Uncallable()
+// @ requires  len(auth) == 16
+// @ requires  sl.AbsSlice_Bytes(buffer, 0, len(buffer))
+// @ preserves acc(s.Mem(ub), R20)
+// @ preserves acc(sl.AbsSlice_Bytes(ub, 0, len(ub)), R20)
+// @ preserves acc(sl.AbsSlice_Bytes(auth, 0, len(auth)), R30)
+// @ ensures   reserr != nil ==> reserr.ErrorMem()
+// @ ensures   reserr != nil ==> sl.AbsSlice_Bytes(buffer, 0, len(buffer))
+// @ ensures   reserr == nil ==>
+// @ 	sl.AbsSlice_Bytes(res, 0, len(res)) &&
+// @ 	(sl.AbsSlice_Bytes(res, 0, len(res)) --* sl.AbsSlice_Bytes(buffer, 0, len(buffer)))
```

Comment:
cannot be proven, even though we are able to assert this wand right at the end of the method